uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/api/v3/LeaveRequestTest.php

```diff
@@ -921,11 +921,13 @@ public function testGetFullShouldNotIncludeTheBalanceChangeIfTheReturnOptionIsNo
     $expectedValues = [
       [
         'id' => $leaveRequest1->id,
-        'dates' => $this->createLeaveRequestDatesArray($leaveRequest1)
+        'dates' => $this->createLeaveRequestDatesArray($leaveRequest1),
+        'contact_id' => $leaveRequest1->contact_id
```

Comment:
We cannot change the API output like that. As I said on another comment, we need to remove the contact_id from the response if it was not specified